apps/ledger-live-desktop/src/logger/logger-transport-renderer.js

```diff
@@ -1,37 +1,2 @@
 import MemoryTransport from "./logger-transport-in-memory";
-import Transport from "winston-transport";
-import { getEnv } from "@ledgerhq/live-common/env";
-
-export default class RendererTransport extends Transport {
-  ipcRenderer = require("electron").ipcRenderer;
-
-  constructor(...args) {
-    super(...args);
-    this.memory = new MemoryTransport(...args);
-  }
-
-  getMemoryLogs() {
-    return this.memory.getMemoryLogs();
-  }
-
-  log(info, callback) {
-    setImmediate(() => {
-      this.emit("logged", info);
-    });
-
-    // In case of experimental execution on renderer, we actually forward this whole action to memory
-    if (getEnv("EXPERIMENTAL_EXECUTION_ON_RENDERER")) {
-      this.memory.log(info, callback);
-      return;
-    }
-
-    try {
-      this.ipcRenderer.send("log", { log: info });
-    } catch (e) {
-      // a malformed tracking data can cause issues to arise better fail safe this
-      console.error(e, info);
-    }
-
-    callback();
-  }
-}
+export default MemoryTransport;
```

Comment:
i'm not sure to understand what you meant? 

in the previous code, and in the case of EXPERIMENTAL_EXECUTION_ON_RENDERER, the previous class was a passthrough to the underlying memory implementation. we keep the logger-transport-{thread}.js structure at the moment as they are entry point for the different threads. maybe in a near future, this can be simplified even more but at least this code was easy dead code here.

i'm going to double check Ctrl+E still works properly